deps/chakrashim/src/v8v8.cc

```diff
@@ -18,11 +18,15 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 // IN THE SOFTWARE.
 
+#include <algorithm>
```

Comment:
It is needed by `std::remove` used in this file. I moved it to front (recall cpplint complains about similar styles).